packages/babel-standalone/Gulpfile.js

```diff
@@ -61,14 +63,15 @@ function webpackBuild(filename, libraryName, version) {
     plugins: [
       new webpack.DefinePlugin({
         "process.env.NODE_ENV": '"production"',
-        BABEL_VERSION:
-          JSON.stringify(require("babel-core/package.json").version),
+        BABEL_VERSION: JSON.stringify(
+          require("babel-core/package.json").version,
```

Comment:
This is still defining the `BABEL_VERSION` by loading the `babel-core` from `node_modules` instead of the `babel-core` being compiled 🤔